prow/external-plugins/cherrypicker/server.go

```diff
@@ -358,7 +358,9 @@ func (s *Server) handlePullRequest(l *logrus.Entry, pre github.PullRequestEvent)
 			if targetBranch == baseBranch {
 				resp := fmt.Sprintf("base branch (%s) needs to differ from target branch (%s)", baseBranch, targetBranch)
 				l.Info(resp)
-				s.createComment(l, org, repo, num, ic, resp)
+				if err := s.createComment(l, org, repo, num, ic, resp); err != nil {
+					l.WithError(err).Errorf("Failed to create comment on %s/%s#%d: %q.", org, repo, num, resp)
```

Comment:
Please use fields instead of printf-style logging. There are already fields for org, repo and num, so at most one for `resp` is needed

Reply:
Fixed.